### DISCUSSION TITLE
Something that helps de-identification of data, or scores identifability

### DISCUSSION THREAD

**Discussion by joshwlambert** [WHO-Collaboratory/collaboratory-epipipeline-community#12] [Ideas]
WHO Collaboratory Hackathon Idea.

**Comment by ntncmch** [comment]
See R package scdMicro: http://sdctools.github.io/sdcMicro/ which comes with a Shiny app and lots of documentation: https://sdcpractice.readthedocs.io/en/latest/sdcMicro.html
